### PR TITLE
styling start learning button.

### PR DESCRIPTION
**Pull request by ryonryon**
@axross Can you check the change?

### CODE REVIEW

File: lib/src/widgets/dashboard_screen/start_learning_floating_action_button.dart [hunk 1]
```diff
@@ -5,12 +5,12 @@ import 'package:saraka/constants.dart';
...
   Widget build(BuildContext context) => FloatingActionButton.extended(
-        heroTag: null,
-        backgroundColor: SarakaColors.lightBlue,
+        backgroundColor: SarakaColors.lightRed,
+        foregroundColor: SarakaColors.white,
         icon: Icon(Feather.getIconData('play')),
         label: Text(
           'Start learning',
-          style: SarakaTextStyles.buttonLabel,
+          style: SarakaTextStyles.buttonLabel.copyWith(letterSpacing: 1),
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/start_learning_floating_action_button.dart]: Why did you change the style to be `letterSpacing: 1` here?

Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/start_learning_floating_action_button.dart]: as keeping consistency. I followed the "New card" button.

Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/start_learning_floating_action_button.dart]: Deleted.

**Review by axross — APPROVED** [review]
Thanks!
